OpenRastaSwagger/Discovery/Heuristics/DiscoverHttpMethodVerbs.cs

```diff
@@ -23,6 +24,20 @@ public bool Discover(MethodInfo publicMethod, OperationMetadata methodMetdata)
             if (methodAttribute != null)
             {
                 methodMetdata.HttpVerb = methodAttribute.Method;
```

Comment:
OpenRastaSwagger selects potential Handlers to match the Uri, then scans all of the methods in that handler for matches. The issue in this case is that if there's a Name on the OpenRasta configured Uri, and a name configured on the Handler method, at no point does OpenRastaSwagger check to see if they match, so identifies false positives for handler methods